File: proyecto/src/Bank.java
import java.util.ArrayList;

public class Bank {
    private ArrayList <Employee> employee = new ArrayList<Employee>();

    

    

    public void addEmployees(String name, int age, double salary, long accountNumber, char typeAccount) {
        employee.add(new Employee(name, age, salary, accountNumber, typeAccount));
    }

    public void addBankAcc(int i, long bankAccount, char type) {
        employee.get(i).addBankAcc(bankAccount, type);
    }
    public ArrayList<Employee> getEmployee() {
        return employee;
    }
}

File: Bank.py
class bank:
    def __init__():
        self.employee = []

    def add_employees(self, name, age, salary, account_number, type_account):
        self.employee.append(Employee(name, age, salary, account_number, type_account))

    def add_bank_acc(self, i, bank_account, type):
        self.employee[i].add_bank_acc(bank_account, type)

    def get_employee(self):
        return self.employee